test/rootful.bats

```diff
@@ -38,3 +38,25 @@ skipIfDockerUnavailableAndNotRunAsRoot() {
 	skipIfDockerUnavailableAndNotRunAsRoot
 	testPortForwarding -u root:root -v "$PODMAN_ROOT_DATA_DIR:/var/lib/containers/storage" "${PODMAN_IMAGE}"
 }
+
+@test "$TEST_PREFIX quedlet - generate service" {
+	if [ "${TEST_SKIP_QUADLET:-}" = true ]; then
+		skip "TEST_SKIP_QUADLET=true"
+	fi
+	$DOCKER run --rm -u podman:podman \
+		-v "$BATS_TEST_DIRNAME/quadlet/hello_world.container:/etc/containers/systemd/hello_world.container" \
+		--pull=never "${PODMAN_IMAGE}" \
+		/usr/local/libexec/podman/quadlet -dryrun > /tmp/test.service # this goes to tmp because we are not root below
```

Comment:
Can you softlink the `quadlet` binary within `/usr/local/bin/quadlet` and make the test access it via its short binary name `quadlet` as opposed to using the whole path, given that users would also use it that way, wouldn't they?
(Same applies to the test within `rootless.bats`, of course.)

Reply:
Hi, not users do not use quadlet on their own. It is systemd generator used by systemd.